en/deploy-on-general-kubernetes.md

```diff
@@ -10,72 +10,71 @@ This document describes how to deploy a TiDB cluster on general Kubernetes.
 - For the demonstration environment or functional verification, you can use network storage, such as `ebs` and `nfs`.
 
-Different components of a TiDB cluster have different disk requirements. Before deploying a TiDB cluster, select the appropriate storage class for each component according to the storage classes supported by the current Kubernetes cluster and usage scenario. You can set the storage class by modifying `storageClassName` of each component in `values.yaml`. For the [storage classes](configure-storage-class.md) supported by the Kubernetes cluster, check with your system administrator.
+Different components of a TiDB cluster have different disk requirements. Before deploying a TiDB cluster, select the appropriate storage class for each component according to the storage classes supported by the current Kubernetes cluster and usage scenario.
+
+You can set the storage class by modifying `storageClassName` of each component in `<cluster-name>/tidb-cluster.yaml` and `<cluster-name>/tidb-monitor.yaml`. For the [storage classes](configure-storage-class.md) supported by the Kubernetes cluster, check with your system administrator.
 
 > **Note:**
 >
 > If you set a storage class that does not exist in the TiDB cluster that you are creating, then the cluster creation goes to the Pending state. In this situation, you must [destroy the TiDB cluster in Kubernetes](destroy-a-tidb-cluster.md).
 
 ### Cluster topology
 
-The deployed cluster topology by default has 3 PD Pods, 3 TiKV Pods, 2 TiDB Pods, and 1 Monitor Pod. In this deployment topology, the scheduler extender of TiDB Operator requires at least 3 nodes in the Kubernetes cluster to provide high availability. If the number of Kubernetes cluster nodes is less than 3, 1 PD Pod goes to the Pending state, and neither TiKV Pods nor TiDB Pods are created.
-
-When the number of nodes in the Kubernetes cluster is less than 3, to start the TiDB cluster, you can reduce both the number of PD Pods and TiKV Pods in the default deployment to `1`, or modify the `schedulerName` in `values.yaml` to `default-scheduler`, a built-in scheduler in Kubernetes.
+The deployed cluster topology by default has 3 PD Pods, 3 TiKV Pods, and 2 TiDB Pods. In this deployment topology, the scheduler extender of TiDB Operator requires at least 3 nodes in the Kubernetes cluster to provide high availability.
 
-> **Warning:**
->
-> `default-scheduler` is only applicable to the demonstration environment. After `schedulerName` is modified to `default-scheduler`, the scheduling of TiDB clusters neither guarantees high availability of data nor supports features such as [TiDB stable scheduling](https://github.com/pingcap/tidb-operator/blob/master/docs/design-proposals/tidb-stable-scheduling.md).
+If the number of Kubernetes cluster nodes is less than 3, 1 PD Pod goes to the Pending state, and neither TiKV Pods nor TiDB Pods are created.
```

Comment:
```suggestion
If the number of Kubernetes cluster nodes is less than 3, 1 PD Pod will go to the Pending state, and neither TiKV Pods nor TiDB Pods will be created.
```

Reply:
@DanielZhangQD I suggest we use the present tense as much as possible and limit the use of "will". Please refer to [Present tense  |  Google developer documentation style guide](https://developers.google.cn/style/tense?hl=en)

Reply:
I suggest we use the present tense as much as possible and limit the use of "will". Please refer to [Present tense  |  Google developer documentation style guide](https://developers.google.cn/style/tense?hl=en)